content/pt-br/docs/reference/setup-tools/kubeadm/_index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Kubeadm"
+weight: 10
+no_list: true
+content_type: conceito
+card:
+  name: reference
+  weight: 40
+---
+
+<img src="/images/kubeadm-stacked-color.png" align="right" width="150px">O Kubeadm é uma ferramenta criada para fornecer o `kubeadm init` e o `kubeadm join` como "caminhos rápidos" de melhores práticas para criar clusters Kubernetes.
+
+O kubeadm executa as ações necessárias para colocar um cluster minimamente viável em funcionamento. Projetado para se preocupar apenas com a inicialização e não com o provisionamento de máquinas. Da mesma forma, a instalação de vários complementos úteis, como o Kubernetes Dashboard, soluções de monitoramento e complementos específicos da nuvem, não está no escopo.
+
+Em vez disso, esperamos que ferramentas de alto nível e mais personalizadas sejam construídas em cima kubeadm e, idealmente, usando o kubeadm como base de todas as implantações tornará mais fácil a criação de clusters em conformidade.
```

Comment:
Missing "do"/"de" after "sejam construídas em cima".
Although the original sentence is "will make it easier", "tornará" is not really idiomatic Portuguese in this sentence. I think "torna" might read and sound better.